docs/reference/dockerd.md

```diff
@@ -62,8 +62,7 @@ Options:
   -G, --group string                          Group for the unix socket (default "docker")
       --help                                  Print usage
   -H, --host list                             Daemon socket(s) to connect to
-      --host-gateway-ip ip                    IP address that the special 'host-gateway' string in --add-host resolves to.
-                                              Defaults to the IP address of the default bridge
+      --host-gateway-ip list                  IP addresses that the special 'host-gateway' string in --add-host resolves to. Defaults to the IP addresses of the default bridge
```

Comment:
Oh! Can you manually wrap / format this one? This page is still manually maintained and this block rendered as a code-block, so wrapping reduces horizontal scrolling; see https://docs.docker.com/reference/cli/dockerd/

Reply:
Will do ... I did that at first, then thought maybe it was supposed to exactly match the `--help` output. Doh!

`host-gateway-ip` isn't a new option, but it's not in the manpage in this repo or moby ... I'll add it.

Reply:
Yeah we really need to;

- update our packaging to use the man-page from moby/moby instead (and remove the man page from this repo)
- look at moving the dockerd reference docs to moby/moby, and use the "docs-generator" to generate these flag-descriptions and the "YAML" docs so that we can use the same formatting as the other reference docs.